src/content/blog/astro-530.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Astro 5.3'
+description: 'Astro 5.3 is here with faster page rendering, easier setup for experimental sessions, and more!'
+homepageLink:
+  title: 'Astro 5.3'
+  subtitle: 'Available now!'
+publishDate: '2025-02-13'
+authors:
+  - matt
+coverImage: '/src/content/blog/_images/astro-530/blog-post-5-3.webp'
+socialImage: '/src/content/blog/_images/astro-530/og-astro-5-3.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+
+**Astro 5.3 is here with faster page rendering, easier setup for experimental sessions, and more!**
+
+💘 It's the season of love, and you're sure to fall for these improvements to Astro:
+
+- [**Faster page rendering**](#faster-page-rendering)
+- [**Automatic session storage setup**](#automatic-session-storage-setup)
+- [**More control over Netlify bundling**](#more-control-over-netlify-bundling)
+- [**Improved handling of `HEAD` requests**](#improved-handling-of-head-requests)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Faster page rendering
+
+Astro makes it easy to use async functions inside components, but previously this meant that every component and page was rendered asynchronously, even if it wasn't needed. When this happens for every component on a page, the overhead from these promises can really add up!
+
+Now, thanks to an awesome community contribution from [MatthewLymer](https://github.com/MatthewLymer) from [Fora](https://fora.com), Astro will only render components asynchronously if... (a)wait for it... they actually contain an async function call.
+
+In benchmarks, this has resulted in a significant speedup for rendering synchronous components. This benefits both on-demand and static builds, though the biggest improvements are seen in SSR response times. This even helps pages that use async functions, because most components on the page will still be synchronous.
+
+The exact amount of the improvement will depend on the individual sites. Complex pages with many components will see the biggest improvements, while pages that use slower external APIs at request time may be unchanged. In our testing, SSR responses are generally 1.5-2x faster unless the page is querying external APIs. Some examples:
+
+- For the 5300-page [Astro docs site](https://docs.astro.build), it **cut static build times by 10-15%.**
+- In the Platformic SSR benchmark, which generates a page with over 3000 components, requests are **4x faster**.
```

Comment:
Do we have a link to the benchmark? (not necessarily the results, just the source)
```suggestion
- In the Platformic SSR benchmark, which generates a page with over 3000 components, requests are **4x faster**.
```

Reply:
Great catch on the name, Reuben! I'd be tempted to link the name, too.

Reply:
I'm happy with adding the link to the benchmark suite, can you add it @TheOtterlord ?